### ISSUE TITLE
Badass Project Review Updates!

### ISSUE THREAD

**Issue by harmanTheCarMan** [LearnersGuild/los#50]
### Expected vs. Actual

_What were you attempting to do? What actually happened?_
I was trying to review projects at the end of the cycle. I was able to review projects but it was hard to keep track of all of the projects which I had reviewed and which I still needed to review for the week. 

I was expecting it to be easy to search for my output in the echo chat to see which groups I had voted for however the summaries of my voting were clouded by all of the other voting members' messages of having voted on a particular project ("User X/26 has voted on project #project1 blah blah" (repeated 25 more times))
### Steps to Reproduce

_How can you reproduce the bug? Include as much detail as you can._
-Try to review many projects at the end of the week in a short amount of time before the new cycle starts. 

-Also try and keep track of all projects you have voted on already including linking group names (i.e. enthusiastic-hawk) to their actual team or goal name (goal 60) and or artifact links to be able to see their code and properly review for quality and completeness. 

-Being unable to track all of these different projects causes less data collection from all learners since they are not able to review all projects. 
### Environment and Versions

_What versions of relevant external software were you using (browser, OS, etc.)?_
Chrome in Mac OS
### What is a Solution?

Create a dropdown or sidebar like "retro sidebar" for Project Reviews:
-Permanently show all teams in review for the week on top of the oakland channel during the review time or if someone uses a command such as  "/review -all" (like how the sidebar comes out)
-can we have a way to show all teams for the week including their #hashtag names, artifact links, and or other resources that each team might want to show for review. 
-Checklist that automatically shows that a project has been reviewed or not by each user
-Even after review, the artifact should still be accessible from #hashtag name or some other format
-Currently, clicking on the #hashtag takes us to the group chat in echo, however a more effective use of it for review would cause clicking on the #hash name to go directly to their github or project artifact. 
-Track who has not yet finished voting or amount of users still needed to vote on project, could be as simple at "21/24" 
-For Shereef or another moderator, it would be useful if it shows the one or two users who may still need to vote on the moderator's echo console. This way moderator can easily close the session when all is said and done.

**Comment by heyheyjp** [comment]
Would like to take a bit more time to really digest it all but wanted to say quickly that this is great, @harmanlearns. Thank you!

/cc @shereefb @tannerwelsh

**Comment by jeffreywescott** [comment]
This is awesome, @harmanlearns ... also, have you seen `/project list -r`?

**Comment by harmanTheCarMan** [comment]
Yes I have! I believe James Stewart mentioned it around the time I believe you added it. It also works as a hack :) This issue had actually started to be written before that was implemented and even before the last voting session where we only voted on one main project and our own project so it was easier to track. However I still wanted to submit this as an issue if were to continue reviewing as many of the projects as possible.

-Harman
